app/models/restaurant.rb

```diff
@@ -15,7 +15,7 @@
 #
 
 class Restaurant < ApplicationRecord
-  validates :name, :cuisine, :rating, :address, :max_time, :kosher, presence: true
+  validates :name, :cuisine, :rating, :address, :max_time, presence: true
   validates :rating, numericality: true
```

Comment:
Instead I added in the migration null: false (otherwise it forced me to write kosher: true in the seeds)